akka-docs/src/main/paradox/java/multi-jvm-testing.md

```diff
@@ -0,0 +1 @@
+../scala/multi-jvm-testing.md
```

Comment:
This is to avoid paradox errors as follows in the java directory:

>No reference link corresponding to multi-jvm-testing.md

It's not referenced from `java/index.md` so adding this has no impact in the resulting pages.